.github/ISSUE_TEMPLATE/product-idea.md

```diff
@@ -0,0 +1,38 @@
+---
+name: product-idea
+about: Share a new market-produkt idea that would benefit you.
+title: [name] product
+labels: ''
+assignees:
+  - ivy-rew
+  - alexsuter
+  - ivy-lmu
+  - weissreto
+  - ivy-bb
+  - axonivy-market
```

Comment:
axonivy-market?

Reply:
maybe only one or two guys should be the gatekeeper?

Reply:
remove axonivy-market. 

And yes, we should define the responsibilities also internally. Now we have a growing eco-system and nobody is really responsible. Despite personal efforts :)